web/package.json

```diff
@@ -12,7 +12,8 @@
     "core-js": "^3.8.3",
     "postcss": "^8",
     "tailwindcss": "^3",
-    "vue": "^3.2.13"
+    "vue": "^3.2.13",
+    "vue-router": "^4.0.16"
```

Comment:
For the dsprotection frontend I used version 4.0.13, not sure if that is a problem

Reply:
We should be fine, these are only minor changes that should not break anything